tests/system/action/motion/test_follow_recommendation.py

```diff
@@ -31,6 +31,7 @@ def setUp(self) -> None:
                 "state_id": 77,
                 "recommendation_id": 76,
                 "recommendation_extension": "test_test_test",
+                "submitter_ids": [],
```

Comment:
You probably added this because the test failed otherwise, right? Please adjust the code then to be able to handle unset fields, since we have no guarantee that empty array fields are actually set to `[]`- they might as well be emtpy.

Reply:
Allow the submitter_ids field to be empty in the permissions check.